src/zap/zapimage.cpp

```diff
@@ -1547,7 +1547,7 @@ void ZapImage::OutputTables()
 
 #if defined(FEATURE_PAL)
     // PAL library requires native image sections to align to page bounaries.
-    SetFileAlignment(0x1000);
+    SetFileAlignment(0x10000);
```

Comment:
> How are the native binary formats like ELF dealing with the 64k pages? Do they have the sections aligned at 64k boundaries as well?

@jkotas We objdumped an elf.  The sections are only aligned to something like 8 byte boundaries.  The VA of the page is incremented by a multiple of the maximum page size.  I think for Arm64 is should be 16byte boundaries.

I took a look at "Visual Studio, Microsoft Portable Executable and Common Object File Format Specification Revision 9.3 – December 29, 2015"  the following statement appears in the text.

```
If the SectionAlignment is less than the architecture’s page size, then FileAlignment 
must match SectionAlignment.
```

I think this would allow the PE to behave similarly to ELF.